modules/cert-manager-configure-cloud-credentials-aws-non-sts.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-aws.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-aws-non-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} on AWS
+
+To configure the cloud credentials for the {cert-manager-operator} on the AWS cluster you must create a `CredentialsRequest` object and allow the Cloud Credential Operator to generate the cloud credentials secret.
+
+.Prerequisites
+
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Create a `CredentialsRequest` resource YAML file, for example, `sample-credential-request.yaml`, as follows:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: cert-manager
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - action:
+      - "route53:GetChange"
+      effect: Allow
+      resource: "arn:aws:route53:::change/*"
+    - action:
+      - "route53:ChangeResourceRecordSets"
+      - "route53:ListResourceRecordSets"
+      effect: Allow
+      resource: "arn:aws:route53:::hostedzone/*"
+    - action:
+      - "route53:ListHostedZonesByName"
+      effect: Allow
+      resource: "*"
+  secretRef:
+    name: aws-creds
+    namespace: cert-manager
+  serviceAccountNames:
+  - cert-manager
+----
++
+[NOTE]
+====
+The use of the `dns.admin` role in the `sample-credential-request.yaml` file is for convenience. To ensure cert-manager runs
+under the least privilege service account, you must create a custom role with the following permissions:
+
+* `dns.resourceRecordSets.*`
+* `dns.changes.*`
+* `dns.managedZones.list`
+====
+
```

Comment:
These permissions are for GCP cloudDNS, not for AWS route53.

Reply:
Ah, found it, Swarup's https://github.com/openshift/openshift-docs/pull/59907#issuecomment-1570414873 is good reminder, but might have been commented in wrong PR, sorry I did not point this out with available time : ).
It is intended to be better added in another GCP PR https://github.com/openshift/openshift-docs/pull/60433 which is already merged.